crates/rolldown_plugin_replace/tests/form/process_check/artifacts.snap

```diff
@@ -8,7 +8,7 @@ source: crates/rolldown_testing/src/integration_test.rs
 ```js
 
 //#region input.js
-if (typeof process !== "undefined" && typeof process.env === "object" && true) {
+{
```

Comment:
It would be better to disable DCE for this test, so can see the actual output from replace plugin, but I couldn't figure our how to do that. In Rollup, `treeshake: false` does that, but `treeshake: TreeshakeOptions::Boolean(false)` seemed to have no effect.

Reply:
@IWANABETHATGUY cc

Reply:
Ok then we should align with rollup

Reply:
Thanks for doing that (#2335).